src/jwplayer/types.d.ts

```diff
@@ -26,9 +26,11 @@ export interface JWPlacementApi {
 
 export type PlayerConfig = {
 	playlist?: Playlist;
+	playlistId?: string | null;
 	playlistUrl?: string;
```

Comment:
Do we still need playlistUrl?

Reply:
Probably not as it seems it's been added with the strategy rules wrapper :)